packages/jsx-email/src/components/button.tsx

```diff
@@ -101,14 +117,16 @@ export const Button: JsxEmailComponent<ButtonProps> = ({
             <span
               dangerouslySetInnerHTML={{
                 __html: `<!--[if mso]>
-            <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" style="height:${height}px;v-text-anchor:middle;width:${width}px;" arcsize="${arcsize}%" ${
+            <v:roundrect ${
+              rel ? `rel="${rel}" ` : ''
+            }href="${href}" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" style="height:${height}px;v-text-anchor:middle;width:${width}px;" arcsize="${arcsize}%" ${
                   borderColor ? `strokecolor=${borderColor}` : ''
                 } ${borderSize ? `strokeweight="${borderSize}px"` : `stroke="false"`} ${
                   backgroundColor ? `fillcolor=${backgroundColor}` : `fill="false"`
                 }>
             <w:anchorlock/>
             <center style="font-size:${fontSize}px;${textColor ? `color:${textColor};` : ''}">
-            ${children}
+            ${vmlFallbackLinkContents}
```

Comment:
I have never interpolated a `ReactNode` into a string like is done with `children` here but I assume that pattern works given that this is what already was here.